src/gui/attributetable/qgsdualview.h

```diff
@@ -358,4 +358,24 @@ class GUI_EXPORT QgsAttributeTableMapLayerAction : public QAction
     QModelIndex mFieldIdx;
 };
 
+class GUI_EXPORT QgsAttributeTableActionCopyCellContent : public QAction
```

Comment:
Why a QAction subclass? Couldn't you just have a standard QAction with the execute (or "copyCellContents") slot in QgsDualView itself?